### PR TITLE
Etherscan link

### LINKED ISSUE #30: [Features] Add etherscan link to epoch key
Can add an arrow in the table row

### PR DESCRIPTION
**Pull request by kittybest**
- let `NETWORK` become a list of objects containing `explorer`, `provider`, and `name`.
- add `etherscan` button on each `event-card`

close #30

### CODE REVIEW

File: packages/frontend/src/components/EpochKeyEvent.jsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default observer(({ attestation }) => {
         )}
       </div>
       <p style={{ minWidth: '100px', fontSize: '12px' }}>
-        {dayjs(timestamp * 1000).from(dayjs())}
+        {dayjs(+blockTimestamp * 1000).from(dayjs())}
       </p>
     </div>
```
Comment (CJ-Rose) [inline packages/frontend/src/components/EpochKeyEvent.jsx:40]: add arrow and link here as well?

Comment (kittybest) [reply packages/frontend/src/components/EpochKeyEvent.jsx:50]: yep, thanks for all the above comments, updated!

**Comment by kittybest** [comment]
@vivianjeng  please help me to fill out the provider url, thanks!

File: packages/frontend/src/components/LastDeploymentCard.jsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export default observer(() => {
               ui.isMobile
             )}
             <a
-              href={`https://goerli.arbiscan.io/address/0x${BigInt(
+              href={`${info.network.explorer}/address/0x${BigInt(
```
Comment (vivianjeng) [inline packages/frontend/src/components/LastDeploymentCard.jsx:33]: this should be padding to length 40 
```ts
0x${BigInt(attesterId).toString(16).padStart(40, '0')}
```

Comment (kittybest) [reply packages/frontend/src/components/LastDeploymentCard.jsx:33]: why we need to padStart?

Comment (vivianjeng) [reply packages/frontend/src/components/LastDeploymentCard.jsx:33]: 0x0023.. should be different from 0x23..
If it is not padding 0s

**Review by CJ-Rose — APPROVED** [review]

File: packages/frontend/src/components/LastDeploymentCard.jsx (hunk 2 above)
Comment (kittybest) [reply packages/frontend/src/components/LastDeploymentCard.jsx:33]: Only the href link need to padStart to lead to correct explorer page, right?
For the display, if there's so many 0, then after `shortenId`, they will look so alike (e.g.`0x00...123`)

Comment (vivianjeng) [reply packages/frontend/src/components/LastDeploymentCard.jsx:33]: yes

**Pull request merged**